### PR TITLE
adds formly-skip-ng-model-attrs-manipulator attribute option and skip be selector option

### PR DESCRIPTION
**Pull request by douglas-mason**

### CODE REVIEW

File: src/run/formlyNgModelAttrsManipulator.js [hunk 1]
```diff
@@ -18,7 +18,10 @@ function addFormlyNgModelAttrsManipulator(formlyConfig, $interpolate) {
       return template;
     }
     el.innerHTML = template;
-    var modelNodes = el.querySelectorAll('[ng-model], [data-ng-model]');
+    var elementFilter = ':not([formly-skip-ng-model-attrs-manipulator])' +
+                        ':not(' + data.skipNgModelAttrsManipulator + ')';
```
Comment (kentcdodds) [inline src/run/formlyNgModelAttrsManipulator.js]: Awesome! I should have covered this in the tests, but what if I pass `false ` as the skipNgModelAttrsManipulator? :-)

**Comment by douglas-mason** [comment]
@kentcdodds updated

**Comment by kentcdodds** [comment]
Looks good to me! :-)

**Comment by kentcdodds** [comment]
Hmmmm... Can't be auto merged... I'll have to look at it when I'm at my machine.

**Comment by douglas-mason** [comment]
looks like you updated the tests after I forked it, I'll fix my fork

**Comment by douglas-mason** [comment]
@kentcdodds  merge conflict fixed :)

**Comment by kentcdodds** [comment]
Awesome! Thanks! I'll release this soon!

**Comment by kentcdodds** [comment]
This has been officially released in `6.16.0`. Thanks again!

**Comment by douglas-mason** [comment]
awesome!  you're welcome, thanks for accepting

**Comment by kentcdodds** [comment]
Hey @douglas-mason, I'm writing a blogpost that references this PR, do you mind if I reference you in it? https://medium.com/@kentcdodds/78281ea47455

**Comment by kentcdodds** [comment]
Linking this to #358

**Comment by douglas-mason** [comment]
Sure , thanks !

**Pull request merged**